styles/custom.scss

```diff
@@ -8,54 +8,51 @@ $link-color: #0d3e79;
 $link-decoration: none;
 $hr-color: #dee2e6;
 $hr-opacity: 1;
-$link-decoration: none;
 $nav-shadow: #d5d5d5;
 
+.link {
+  border-bottom: 1px dotted;
+}
+
 .status-filter-inactive {
   border-radius: 0px !important;
   border: none !important;
-  color: #6c757d !important;
+  color: #0d3e79 !important;
   padding-bottom: 5px !important;
 }
 
 .status-filter-active {
   border-radius: 0px !important;
   border: none !important;
-  border-bottom: 5px solid #ff7261 !important;
-  color: #ff7261 !important;
+  border-bottom: 5px solid #0d3e79 !important;
+  font-weight: bold;
+  color: #0d3e79 !important;
 }
 
 .status-filter:hover {
   color: #ff7261 !important;
 }
 
-.chart-toggle-inactive {
-  border-radius: 0px !important;
-  border: none !important;
-  color: #6c757d !important;
+.chart-toggle-disabled {
```

Comment:
this is actually called `disabled` in the code, but maybe i should switch the code to say `inactive` to keep it how it was + the same as the other classes that are called `inactive` ?

Reply:
That sounds good to me. Seems to be the pattern in the rest of the .scss file for sure.